Lib/test/test_zipimport.py

```diff
@@ -712,7 +717,7 @@ def bad_decompress(*args):
             zip_file.writestr('bar.py', b'print("hello world")', ZIP_DEFLATED)
         zi = zipimport.zipimporter(TEMP_ZIP)
         with support.swap_attr(zlib, 'decompress', bad_decompress):
-            self.assertRaises(TypeError, zi.get_source, 'bar')
+            self.assertRaises((TypeError, AttributeError), zi.get_source, 'bar')
```

Comment:
I think in all these cases that if the tests aren't really needed in the Python implementation, then they can be removed.  It would be nice if had some coverage numbers for the module, but sicne we don't really have that infrastructure, I think it's fine to just remove tests that we don't need any more.